vcd/resource_vcd_vapp_vm.go

```diff
@@ -128,6 +128,12 @@ func resourceVcdVAppVm() *schema.Resource {
 				Optional: true,
 				Set:      resourceVcdVmIndependentDiskHash,
 			},
+			"expose_hardware_virtualization": &schema.Schema{
```

Comment:
Why not just call it `hardware_virtualization` or `nested_hypervisor`? Just looking for a clear, but shorter name here.

Reply:
We discussed this item, and the outcome was that we need a name that says we are just setting a switch. We don't want to convey the idea that this operation does more than changing metadata in the entity.

Reply:
Yeah. I wanted to make this shorter as it is long, but otherwise probably it's not fully clear.

Reply:
I agree that purposed shorter names aren't fully clear.

Reply:
After a long discussion... I agree that pure `hardware_virtualization` may be intuitively mixed with virtualization in general and that we should find a keyword that instantly makes it clear what it is, but at the same time is short enough. Hence, new suggestion: `hw_assisted_virtualization`
- It's probably safe to assume that `hw` is a well known abbreviation in our field
- Impact: we can use `hw` abbreviation in other fields in the future too

Reply:
Another approach is to follow vCD API and call it by use case as opposed to by what it's doing:
```
NestedHypervisorEnabled bool   `xml:"nestedHypervisorEnabled,attr,omitempty"` // True if hardware-assisted CPU virtualization capabilities in the host should be exposed to the guest operating system.
```
So, in our case that could be: `nested_hypervisor`

Reply:
Morning is brighter than the evening... After all, I'm OK with current `expose_hardware_virtualization` implementation. It's longest field name we have, but at least it makes things clear!

Reply:
Agreed